go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -248,80 +249,72 @@ func (oc *Controller) addEgressFirewallRules(hashedAddressSetNameIPv4, hashedAdd
 
 // createEgressFirewallRules uses the previously generated elements and creates the
 // logical_router_policy/join_switch_acl for a specific egressFirewallRouter
-func createEgressFirewallRules(priority int, match, action, externalID string) error {
+func (oc *Controller) createEgressFirewallRules(priority int, match, action, externalID string) error {
+	logicalSwitches := []string{}
 	if config.Gateway.Mode == config.GatewayModeLocal {
-		_, stderr, err := util.RunOVNNbctl("--id=@logical_router_policy", "create", "logical_router_policy",
-			fmt.Sprintf("priority=%d", priority),
-			match, "action="+action, fmt.Sprintf("external-ids:egressFirewall=%s", externalID),
-			"--", "add", "logical_router", types.OVNClusterRouter, "policies", "@logical_router_policy")
+		nodes, err := oc.kube.GetNodes()
```

Comment:
we should use the factories informer cache to get nodes

Reply:
Yes, you're right. Will change